pages/common/pt.md

```diff
@@ -0,0 +1,23 @@
+# pt
+
+> Platinum Searcher. A code search tool similar to ag. Respects .gitignore.
+
+- Find files containing "foo" and print the files with highlighted matches:
+
+`pt foo`
+
+- Find files containing "foo" and display count of matches in each file:
+
+`pt -c foo`
+
+- Find files containing "foo" as a whole word and ignore its case:
+
+`pt -wi foo`
+
+- Find "foo" in files with a name matching provided regular expression:
```

Comment:
We can say `Find "foo" in files with a given extension using a regular expression:`